experimental/cache_size.cc

```diff
@@ -0,0 +1,70 @@
+#include <stdlib.h>
```

Comment:
Late to mention this, and feel free to disregard and send a followup PR, but especially since it includes nonlocal files, it might be handy to give compilation instructions here as I'm not totally sure how to do it.

In general files often get file comments describing what they do: https://google.github.io/styleguide/cppguide.html#File_Comments